sass/styles.scss

```
@@ -0,0 +1,505 @@
+$font-sans:'Open Sans',
+sans-serif;
+$blue:#0240f4;
+$square-cell:minmax(100px, min(300px, 33vw));
+$square-cell2:minmax(100px, min(300px, 16vw));
+
+* {
+    box-sizing: border-box;
+}
+
+//gneral text
+
+body {
+    background-color: darkgray;
+    color: $blue;
+    font-family: $font-sans;
+    font-size: 14px;
+    box-shadow: 30px 50px 10px grey;
+    margin: 0;
+}
+
+div {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+h1 {
+    font-size: 160px;
+    line-height: 1.3;
+    margin: 0;
+}
+
+h2 {
+    align-items: baseline;
+    font-size: 160px;
+    line-height: 0.8;
+    margin: 0;
+}
```

I would think about combining some of the styles here:
```suggestion
h1, h2 {
    font-size: 160px;
    margin: 0;
}

h1 {
    line-height: 1.3;
}

h2 {
    align-items: baseline;
    line-height: 0.8;
}
```

This makes it more clear what is similar and different between these elements.